gradle.properties

```diff
@@ -16,7 +16,7 @@ apacheCommonsCompressVersion=1.18
 apacheCuratorVersion=4.0.1
 apacheZookeeperVersion=3.5.5
 checkstyleToolVersion=8.23
-bookKeeperVersion=4.7.3
+bookKeeperVersion=4.9.2
```

Comment:
I think this may be unintended

Reply:
With the client `4.7.3` I experienced many issues, and as soon as I upgraded it worked fine. As this is going to `master`, we can wait until merging #4394.